internal/pkg/manifest/storage.go

```diff
@@ -154,7 +153,10 @@ func (e *EFSVolumeConfiguration) unsetUIDConfig() {
 
 func (e *EFSVolumeConfiguration) isValid() error {
 	if !e.EmptyBYOConfig() && !e.EmptyUIDConfig() {
-		return errInvalidEFSConfiguration
+		return &errFieldMutualExclusive{
+			firstField:  "uid/gid",
+			secondField: "id/root_dir/auth",
```

Comment:
Do think we should take a list for the inputs instead?
`"cannot specify uid, gid with any of id, root_dir, auth"`

Reply:
Hmm I think I still prefer having
`must specify one, not both, of "uid/gid" and "id/root_dir/auth"`
which makes the field stand out and more explicit to users? However, this does remind me that we need to make sure they specify both `uid` and `gid`.